start_esp/start_esp.py

```diff
@@ -604,7 +604,7 @@ def make_argparser():
         help='''Enable HSTS (HTTP Strict Transport Security).
         ''')
 
-    parser.add_argument('--enable_debug', action='store_true',
+    parser.add_argument('--enable_debug',default=True, action='store_true',
```

Comment:
why do you have this change?

Reply:
Because your PR containing this change caused failure but the log is distained, I fix the log and wanna take a look. Now I saw this presubmit failure was just because with the debug=true, the config would have extra lines.